styles/globals.css

```diff
@@ -1,16 +1,32 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat+Alternates:wght@900&family=Paytone+One&family=Quicksand&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
 html,
 body {
   padding: 0;
   margin: 0;
   font-family: -apple-system, BlinkMacSystemFont, Segoe UI, Roboto, Oxygen,
-    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, sans-serif;
+    Ubuntu, Cantarell, Fira Sans, Droid Sans, Helvetica Neue, "Poppins",
+    sans-serif, Quicksand, Paytone One;
 }
 
 a {
   color: inherit;
   text-decoration: none;
 }
 
+:root {
+  --background-color: #e6ffff;
```

Comment:
Maybe it is easier, to set the variables for the colors without "--button-" or "--placeholder-" in front. You could instead name it like Bootstrap does or so.
![grafik](https://user-images.githubusercontent.com/56833783/111812945-e9196080-88d8-11eb-9aab-53d0be49ac25.png)
That could make the reusability easier later.

Reply:
Will do, thanks!